include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
How does it not work for you? (It won't be able to print the content of a vector<bool>, but that's a missing feature I'm willing to live with...)